core/tauri/src/api/rpc.rs

```diff
@@ -148,6 +156,7 @@ pub fn format_callback<T: Serialize, S: AsRef<str>>(
 /// Note that the callback strings are automatically generated by the `invoke` helper.
 ///
 /// # Examples
+/// TODO: better example to explain
```

Comment:
```suggestion
// TODO: better example to explain
```
(this TODO shouldn't be visible on the docs).